wcivf/templates/home.html

```diff
@@ -5,52 +5,50 @@
 {% block og_title %}Who Can I Vote For?{% endblock og_title %}
 
 {% block content %}
-<div class="ds-scope">
-  <div class="ds-stack-smaller">
-    <div class="ds-card ds-text-centered">
-      <div class="ds-card-body">
-        <h4>Find out about candidates in your area</h4>
-        <div class="ds-field">
-          <form method=post>
-            {% csrf_token %}
-              {{ form|dc_form }}
-            {% if request.GET.invalid_postcode %}
-            <p>
-              <strong>Sorry, we don't know the postcode {{ request.GET.postcode }}.
-                Is there another one you can try?</strong>
-            </p>
-            {% endif %}
-            <button type="submit">Find your candidates</button>
-          </form>
-        </div>
-      </div> 
+
+<div class="ds-card">
+  <div class="ds-card-body ds-text-centered">
+    <h4>Find out about candidates in your area</h4>
+    <div class="ds-field">
+      <form method=post>
+        {% csrf_token %}
+          {{ form|dc_form }}
+        {% if request.GET.invalid_postcode %}
+        <p>
+          <strong>Sorry, we don't know the postcode {{ request.GET.postcode }}.
+            Is there another one you can try?</strong>
+        </p>
+        {% endif %}
+        <button class="ds-button-pink" type="submit">Find your candidates</button>
+      </form>
     </div>
+  </div> 
+</div>
 
 {% if upcoming_elections %}
   <div class="ds-card">
     <div class="ds-card-body">
-      <h2>Upcoming elections</h2>
+      <h2>Upcoming by-elections</h2>
```

Comment:
We changed this recently in another PR didn't we? Do we want to change this back?

Reply:
Yes, thanks for catching this.